config/sites/iba.biology.uiowa.edu/config_split.config_split.site.yml

```diff
@@ -0,0 +1,27 @@
+uuid: 97e5f76c-7922-48e6-a935-d3624260db79
+langcode: en
+status: true
+dependencies: {  }
+id: site
+label: iba.biology.uiowa.edu
+description: 'Adding additional person types for Directors, Mentors, Advisory Board, Executive Committee, and Alumni. Add additional fields to profiles for mentor entity reference and program year.'
+weight: 70
+stackable: false
+storage: folder
+folder: ../config/sites/iba.biology.uiowa.edu
+module: {  }
+theme: {  }
+complete_list:
+  - field.field.node.person.field_mentor
+  - field.field.node.person.field_program_year
+  - field.storage.node.field_mentor
+  - field.storage.node.field_program_year
+  - sitenow_people.person_type.advisory_board_member
+  - sitenow_people.person_type.alumni
+  - sitenow_people.person_type.director
+  - sitenow_people.person_type.executive_committee_member
+  - sitenow_people.person_type.mentor
+partial_list:
+  - config_split.config_split.site
+  - 'core.entity_view_display.node.person.*'
```

Comment:
I believe this should be in the complete split list because of https://github.com/uiowa/uiowa/issues/4452.

Reply:
Moved core.entity_view_display.node.person.* to complete split, reexported, committed changes. Thanks you two.

Reply:
>Should the core.entity_form_display.node.person.* also be in the complete split?

I think that is ok to leave as partial.